packages/react/src/components/navigation/navigationDropdownLink/NavigationDropdownLink.module.scss

```diff
@@ -0,0 +1,112 @@
+@import "../navigation.common";
+@value x-small-down, small-down, small-only, medium-only, medium-up, large-only, large-up, x-large-only, x-large-up from "../../../styles/breakpoints.css";
+
+.dropdownLinkItem.dropdownLinkItem {
+  &:focus-within {
+    box-shadow: none;
+  }
+
+  // dropdown menu
+  > div {
+    // remove margins from children
+    * {
+      margin: 0;
+    }
+
+    // dropdown menu item
+    > * {
+      font-weight: normal;
+
+      // dropdown menu item icon
+      span:first-of-type:not(:only-of-type) {
+        align-self: flex-start;
+        margin: 0 var(--spacing-2-xs) 0 calc(var(--spacing-3-xs) * -1);
+      }
+    }
+  }
```

Comment:
Could we use the _.navigation_common.scss to get rid of duplicate CSS with NavigationDropdown.scss?
Maybe this could be a %dropDownItemBase and it could be used like:
```css
> div {
  @extend  %dropDownItemBase;
}
```